src/api/customer/customer-controller.ts

```diff
@@ -15,6 +15,35 @@ const createCustomer = (): CustomerCreationRequestHandler[] => {
     ]
 }
 
+const getCustomer = (): GetCustomersRequestHandler[] =>{
+    return [
+        getCustomersValidator(),
+        paginationSerializer,
+        getCustomersBusiness,
+        getCustomersSerializer,
+        (req, res) => {
+            res.status(OK).json(res.locals.customerToRespond);
+        },
+    ]
+}
+
+const getCustomerUuid = (): GetCustomerByUuidRequestHandler[] =>{
+    return [
+        getCustomerByUuidValidator(),
+        getCustomerByUuidBusiness,
+        getCustomerByUuidSerializer,
+        (req, res) => {
+            if (!res.locals.customerToRespond) {
+                res.status(NOT_FOUND).json(res.locals.customerToRespond);
+            } else {
+                res.status(OK).json(res.locals.customerToRespond);
+            }
```

Comment:
Sugiro deixar de negar a expressão `!`, e inverter a ordem do `if`.
Isso facilita a leitura.

```suggestion
            if (res.locals.customerToRespond) {
                res.status(OK).json(res.locals.customerToRespond);
            } else {
                res.status(NOT_FOUND).json(res.locals.customerToRespond);
            }
```